libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewListener.kt

```diff
@@ -20,27 +25,27 @@ abstract class NavigationViewListener {
     /**
      * Called when NavigationView enters Free Drive state.
      */
-    open fun onFreeDriveStarted() = Unit
+    open fun onFreeDrive() = Unit
```

Comment:
You break API by these changes, is that expected?

Reply:
The interface is marked as experimental, so this is allowed.